packages/rancher-cis-benchmark/charts/templates/configmap.yaml

```diff
@@ -8,7 +8,9 @@ data:
   rke: |-
     <1.16.0: rke-profile-permissive-1.5
     >=1.16.0: rke-profile-permissive-1.6
-  rke2: "rke2-cis-1.5-profile-permissive"
+  rke2: |-
+    <1.20.0: rke2-cis-1.5-profile-permissive
+    >=1.20.0: rke2-cis-1.6-profile-permissive
```

Comment:
The ClusterScanBenchmark above shows  minKubernetesVersion: "1.18.0"
But this change here restricts any version <1.20.0 to rke2-cis-1.5-profile-permissive - why? Should the `minKubernetesVersion` be "1.20.0" then?